tdrs-backend/tdpservice/parsers/util.py

```diff
@@ -320,7 +323,8 @@ def case_aggregates_by_month(df, dfs_status):
             case_numbers = case_numbers.union(curr_case_numbers)
 
         total += len(case_numbers)
-        cases_with_errors += ParserError.objects.filter(case_number__in=case_numbers).distinct('case_number').count()
+        cases_with_errors += ParserError.objects.filter(file=df).filter(
```

Comment:
i snuck this change in here as i was debugging - it should fix the case aggregates counting problem described here 
https://github.com/raft-tech/TANF-app/pull/2643#issuecomment-1734370699
https://github.com/raft-tech/TANF-app/pull/2643#issuecomment-1741360241